setup.py

```diff
@@ -44,7 +44,7 @@ def find_scripts(script_dir, prefix):
     install_requires=[
         "astropy~=4.2",
         'ctapipe>=0.10.4',
-        'ctapipe_io_lst~=0.8.0',
+        'ctapipe_io_lst>=0.8.2',
```

Comment:
Why `>=` here? Do we want to install 0.9 automatically when it comes out?

Reply:
no, I moved it to `~=0.8.2`. Although will this always take `>v0.8.2 & <v0.9`?

Reply:
Yes, `~=0.8.2` means the same as `>=0.8.2` and `<0.9.0a0`, just a more concise way to write it. See https://www.python.org/dev/peps/pep-0440/#compatible-release